code/game/objects/structures/tables_racks.dm

```diff
@@ -45,6 +46,13 @@
 	update_adjacent()
 	..()
 
+/obj/structure/table/proc/checkhealth()
```

Comment:
Uhm, probably because the call to this proc is also on `/obj/structure/table` level?